source/features/edit-readme.tsx

```diff
@@ -38,11 +38,11 @@ async function init(): Promise<void | false> {
 		return;
 	}
 
-	const stickyHeader = readmeHeader.closest('.js-sticky .d-flex');
-	(stickyHeader ?? readmeHeader).after(
+	const headerContainer = readmeHeader.closest('.d-flex.flex-items-center')!;
+	headerContainer.after(
 		<a
 			href={String(url)}
-			className={`${stickyHeader ? 'm-0 p-2' : 'Box-btn-octicon'} btn-octicon float-right`}
+			className={`${headerContainer.parentElement!.classList.contains('js-sticky') ? 'p-2' : 'Box-btn-octicon'} btn-octicon`}
```

Comment:
- `m-0` was added purely to be consistent with GitHub, but I figure it's not necessary after all
- `float-right` doesn't work anyway since there's `flex-justify-between`